test/baselines/adapter_TS_React/admin/src/i18n/de.json

```diff
@@ -0,0 +1,5 @@
+{
+	"test-adapter adapter settings": "Adaptereinstellungen für test-adapter",
```

Comment:
Weird, why is one setting translated and the others use the mock translations?

Reply:
The title is a fixed translation defined in `translations.ts` (it used to be in `words.js.ts`).